packages/tools/src/types/EventTypes.ts

```diff
@@ -116,8 +118,13 @@ type AnnotationAddedEventDetail = {
  * drawing on the viewport.
  */
 type AnnotationCompletedEventDetail = {
```

Comment:
Would like to drop the annotation events to just changed and removed, with the type of change being included in the change type, but that requires more thinking about when it should occur.